### ISSUE TITLE
[Paper Note] Unified Neural Scaling Laws, Ethan Caballero+, arXiv'26, 2026.05

### ISSUE THREAD

**Issue by AkihikoWatanabe** [AkihikoWatanabe/paper_notes#5636]
# URL
- https://arxiv.org/abs/2605.26248
# Authors
  - Ethan Caballero
  - Priyank Jaini
  - David Krueger
  - Irina Rish
# Abstract
  - We present a functional form (that we refer to as a Unified Neural Scaling Law (UNSL)) that accurately models and extrapolates the scaling behaviors of deep neural networks as multiple dimensions all vary simultaneously (i.e. how the evaluation metric of interest varies as one simultaneously varies the number of model parameters, training dataset size, number of training steps, number of inference steps, amount of compute, and various hyperparameters) for various architectures and for each of various tasks within a varied set of upstream and downstream tasks. This set includes large-scale vision, language, math, and reinforcement learning. When compared to other functional forms for neural scaling, this functional form yields extrapolations of scaling behavior that are considerably more accurate on this set.
# Translation (by gpt-5-nano)
- 私たちは、統一ニューラルスケーリング則（UNSL）と呼ぶ関数形を提示します。これは、複数の次元が同時に変化する場合に深層ニューラルネットワークのスケーリング挙動を正確にモデル化し、外挿できるものです。つまり、関心の評価指標が、モデルパラメータ数、トレーニングデータセットのサイズ、トレーニングステップ数、推論ステップ数、計算量、そしてさまざまなハイパーパラメータを同時に変化させた場合に、どのように変化するかを示します。これは、さまざまなアーキテクチャ、および上流タスクと下流タスクの多様な集合に含まれる各タスクに対して適用されます。この集合には、大規模なビジョン、言語、数学、そして強化学習が含まれます。他のニューラルスケーリングの関数形と比較すると、この関数形はこの集合におけるスケーリング挙動の外挿をかなり正確に行います。
# Summary (by gpt-4o-mini)
- UNSLと呼ばれる関数形を提案し、複数の次元が同時に変化する際の深層ニューラルネットワークのスケーリング挙動を正確にモデル化。モデルパラメータ、データセットサイズ、トレーニングおよび推論ステップ数、計算量などが影響を与える様子を示し、大規模なビジョン、言語、数学、強化学習タスクに適用。既存のスケーリング関数と比べ、より精度の高い外挿を実現。

**Comment by AkihikoWatanabe** [comment]
元ポスト:https://x.com/ethancaballero/status/2059686905105563907?s=46&t=Y6UuIHB0Lv0IpmFAjlc2-Q

**Comment by AkihikoWatanabe** [comment]
データセットサイズ、推論ステップ数、幅、深さ、初期化時の重みの標準偏差、学習率、バッチサイズを変数に持ち、それらを同時に変化させても外挿可能なScaling Lawsのようである
